snapshot/snapshot.go

```diff
@@ -608,7 +608,12 @@ func (o *snapshotter) remoteMounts(ctx context.Context, s storage.Snapshot, id s
 		return bindMount(o.upperPath(s.ParentIDs[0])), nil
 	}
 
-	lowerDirOption := fmt.Sprintf("lowerdir=%s", o.lowerPath(id))
+	lowerPath := o.lowerPath(id)
+	if lowerPath == "" {
```

Comment:
Maybe not related to this PR, just a code style suggestion, the error returned from `o.fs.MountPoint` is ignored in `o.lowerPath`, maybe we should return the error from `o.lowerPath` directly.

Reply:
Yes. Let's return the error to containerd